docs/sdk/paying-for-lit/payment-delegation-db.md

```diff
@@ -0,0 +1,269 @@
+- `POST` `/add-users`: This route is used to add users (as Ethereum addresses) as `payees` for a specific `payer`. This allows the `payer` to pay for the usage of Lit for each user, without each user having to own a Capacity Credit
+
+Below we will walk through an example of registering a `payer` and adding users as `payees` for a specific `payer`. The full code implementation can be found [here](https://github.com/LIT-Protocol/developer-guides-code/tree/wyatt/payment-delegation-relayer/payment-delegation-db-relayer/nodejs/src).
+
+## Prerequisites
+
+Before continuing with this guide, you should have an understanding of:
+
+- [Capacity Credits](../capacity-credits.md)
+- Lit's [Chronicle rollup](../../network/rollup.mdx)
+- Lit's [Relayer server](https://github.com/LIT-Protocol/relay-server)
+- You must have a valid Lit Relayer API key
+<!-- TODO Get Google form for applying for API key -->
+  - This can be obtained by filling out this form
+- You should know which paid Lit network you're going to use: `habanero` or `manzano`
+
+This guide doesn't have any external dependencies, but relies on `fetch` being natively available in Node.js, which means the minimum supported version is `v18`.
+
+## Paying for Lit
+
+Like other decentralized networks, Lit has a certain amount of computation available for users that's metered to allow for a responsive network with nodes that are able to stay in-sync with one another.
+
+In order to use Lit, you must reserve capacity on the network. Typically this is done by choosing a `payer` wallet and manually minting a Capacity Credit for it on Chronicle. Afterwards you'd then provide an [Auth Signature](../authentication/auth-sig.md) delegating usage of the Capacity Credit to your users.
+
+This can be difficult to maintain as it often requires a server to be spun up to maintain a database of the `delegatees` and the expirations of their Capacity Credit Delegation Auth Signatures. Additionally, the Delegation Auth Signature, must be attached to every request the `delegatee` submits to the Lit network, requiring the aforementioned server to provide this to the `delegatees` before then can even being to interact with Lit's network.
+
+To simplify this process of delegating Capacity Credits, we've implemented the Payment Delegation Database. Using Lit's Relayer server, you can now register a `payer` wallet (which will mint a Capacity Credit) and manage your `delegatees` using the two simple API routes.
+
+## Registering a Payer Wallet
+
+:::info
+<!-- TODO Get Google form for applying for API key -->
+Registering a payer using Lit's Relayer server requires an API key, if you don't already have one, you can apply for one here.
+:::
+
+:::warning
+After successfully registering a `payer` with the Relayer server, you will receive a `payerSecretKey` as part of the API response. This secret key is essentially the **private key** of your new `payer` wallet and should be treated as any other private key would.
+
```

Comment:
W should add that `API requests using the `payerSecretKey` requires secure context. Should not be used from a browser context`.

Reply:
Added: `API request using the `payerSecretKey` requires secure context, meaning it should not be used from a browser context where it would be leaked to the end user.`